app/models/reservation.rb

```diff
@@ -5,13 +5,13 @@ class Reservation < ApplicationRecord # :nodoc:
   enum status: %i[pending confirmed canceled]
 
   # Associations
-  belongs_to :room
   belongs_to :guest
+  belongs_to :room
   belongs_to :user
 
   # Validations
-  validates :room_id, presence: true
-  validates :guest_id, presence: true
+  validates :status, inclusion: { in: statuses.keys }
```

Comment:
Not really needed in Rails 7.1:
https://blog.saeloun.com/2024/01/02/rails-enums-validate-option/

Reply:
Unfortunately we are not using Rails 7.1 here. Checking the Gemfile, I think we are only using 7.0.8, so this inclusion validation is needed.

Reply:
I'll update the version soon. and Rails 8 is nearly out. : )